### ISSUE TITLE
A11y_AzureDataStudioAAE_ConnecttoExistingController_ColorContrast: The color contrast ratio of error message "ENOENT: no such file or directory open" is lest than 4.5:1

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15776]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.3;#AINotInScope;#ColorContrastAnalyzer;

### Environment Details:
Application Name: Azure Data Studio -AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to "Create new Azure Arc Controller"
4. Tab to "Kube config file path" edit box start
5. Open Color contrast analyzer tool and check the color contrast ratio of error message "ENOENT: no such file or directory open

### Actual:
The color contrast ratio of error message "ENOENT: no such file or directory open" is less than minimum requirement of 4.5:1. It is 3.996:1
Similar issue observed throughout the application where this message is present.

### Expected:
The color contrast ratio of error message "ENOENT: no such file or directory open" should be equal to or greater than minimum requirement of 4.5:1

### User Impact:
Low vision users will not be able to view the message properly.
![15776_A11y_AzureDataStudioAAE_ConnecttoExistingController_ColorContrast](https://user-images.githubusercontent.com/60880975/127480436-776e4ed6-4af9-4d6b-b5f5-4b4b654db062.PNG)

**Comment by Shivamd1** [comment]
@candiceye the issue still repro in in latest azure data studio @Charles-Gagnon 

![image](https://user-images.githubusercontent.com/60880975/150949009-d943a985-abab-4179-90b1-4f4a0f8fbfb2.png)

**Comment by candiceye** [comment]
Used text type property coloring on the error messages, such as in this PR: https://github.com/microsoft/azuredatastudio/pull/16421

Screenshot of the changes:
![image](https://user-images.githubusercontent.com/22386690/154213133-31ec4495-e101-4d8c-852e-f3af899056a9.png)

PR here: https://github.com/microsoft/azuredatastudio/pull/18411

**Comment by Charles-Gagnon** [comment]
@candiceye Apparently this has regressed (or was not properly fixed originally) - could you check that out and see if this is the case?

https://github.com/microsoft/azuredatastudio/pull/16421#issuecomment-1178817318

**Comment by alanrenmsft** [comment]
@candiceye ping...

**Comment by candiceye** [comment]
@Charles-Gagnon  @alanrenmsft  @Shivamd1  
Just tested it out with 1.38.0 insiders and it does not seem to have regressed.
![image](https://user-images.githubusercontent.com/22386690/178813932-844e9a3c-d8c3-499b-925e-efecb9cf0d0a.png)

**Comment by candiceye** [comment]
![image](https://user-images.githubusercontent.com/22386690/178814160-1d4543d8-7691-4814-9c17-3614599766dd.png)

**Comment by candiceye** [comment]
The current color is: #a1260d

**Comment by alanrenmsft** [comment]
I also tried on latest insider build and the color is same as what @candiceye mentioned.
![image](https://user-images.githubusercontent.com/13777222/178814443-a79b3358-a3e5-4ae7-8704-147d78a03791.png)

@Shivamd1 please retest ASAP since the issue is going out of SLA soon.

**Comment by Shivamd1** [comment]
@alanrenmsft @candiceye @candiceye Thanks for the update. verified again and the issue does not repro now closing the bug

![image](https://user-images.githubusercontent.com/60880975/178912484-e20d1cea-71a3-4e90-beac-4e6622ff0438.png)

**Comment by Roopesh-T** [comment]
**GithubTags**:#Closed;
